tests/main/interfaces-accounts-service/task.yaml

```diff
@@ -68,11 +30,21 @@ execute: |
       0 \
       5 'Enabled' 'true' 'EmailAddress' 'test@example.com' 'Name' 'Test User' 'ImapHost' 'imap.example.com' 'SmtpHost': 'mail.example.com'
 
+restore: |
+    session-tool -u test --restore
+
+    apt-get autoremove --purge -y
```

Comment:
Wouldn't this remove more packages than the test actually installed?

Reply:
Hmm, perhaps. I'm open to suggestions on a packaging helper that would let us have precise semantics here.

Reply:
this will not remove gnome-online-accounts itself, no?  that should be done first

Reply:
Perhaps we should revive #7414 which cleans up the packages after the test

Reply:
yes, but we should try not to block this one on that. Also commented about naming there.

Reply:
@zyga I think, this still needed apt-get remove --purge -y gnome-online-accounts